.gitignore

```diff
@@ -288,3 +288,4 @@ win32/*.positions
 win32/ext
 win32/phpts.def
 win32/wsyslog.h
+ext/opcache/shared_alloc_win32.c
```

Comment:
No that is not an intentional change.


Please fix.


Thanks

-M

________________________________
From: Bob Weinand <notifications@github.com>
Sent: Monday, October 5, 2015 4:11:07 PM
To: php/php-src
Cc: Matt Ficken (Insight Global)
Subject: Re: [php-src] OpCache Windows bug fix (#1536)


In .gitignore<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fphp%2fphp-src%2fpull%2f1536%23discussion_r41209590&data=01%7c01%7cv-mafick%40microsoft.com%7c18eec26a3c5742057c8308d2cdda421e%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=ThT9uLdQWHq0vVYTF2YuihW6H95Nsigi0TIhSAhKo6E%3d>:

> @@ -288,3 +288,4 @@ win32/*.positions
>  win32/ext
>  win32/phpts.def
>  win32/wsyslog.h
> +ext/opcache/shared_alloc_win32.c


I guess this .gitignore change isn't intentional here...?

-
Reply to this email directly or view it on GitHub<https://na01.safelinks.protection.outlook.com/?url=https%3a%2f%2fgithub.com%2fphp%2fphp-src%2fpull%2f1536%2ffiles%23r41209590&data=01%7c01%7cv-mafick%40microsoft.com%7c18eec26a3c5742057c8308d2cdda421e%7c72f988bf86f141af91ab2d7cd011db47%7c1&sdata=uIIXcgNhudUPdE6bWh8CZXTHVbT7Ej72lkLjBtdSryc%3d>.